algebra-core/src/biginteger/mod.rs

```diff
@@ -107,8 +111,8 @@ pub trait BigInteger:
     /// Returns true iff this number is zero.
     fn is_zero(&self) -> bool;
 
-    /// Compute the number of bits needed to encode this number. Always a
-    /// multiple of 64.
+    /// Compute the exact number of bits needed to encode this number. Does not need
+    /// to be multiple of 64
     fn num_bits(&self) -> u32;
```

Comment:
Was it always like this? If no, people may be relying on it, so might be better to have a new method.

Reply:
It was always like this, the comments were wrong